tests/distributed/test_against_external_daemon/test_single_instance.py

```diff
@@ -131,7 +131,7 @@ def docker_image():
 
 @pytest.mark.parametrize('silent_log', [True, False])
 @pytest.mark.parametrize('parallels', [1, 2, 3])
-def test_l_r_l_with_upload(silent_log, parallels, docker_image, mocker):
+def test_l_r_l_with_upload_docker(silent_log, parallels, docker_image, mocker):
```

Comment:
the docker `suffix` I would remove. We use docker to "simulate" remote instances but is not  a "docker" or "docker pod" feature we are testing

Reply:
Reason is, that there is another test with the exact same name, but not using docker. I wanted to distinguish them. So rather `_remote`?

Reply:
where is that other test?

Reply:
yes I think rempte shows better the spirit